.github/workflows/phpcodesniffer.yml

```diff
@@ -0,0 +1,23 @@
+name: PHP CodeSniffer
+
+# Controls when the action will run. 
+on: ['push','pull_request']
+jobs:
+  phpcs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # important!
+      - name: Install PHP_CodeSniffer
+        run: |
+          curl -OL https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar
+          php phpcs.phar --version
+
+      - uses: mvr320/action-php-codesniffer@v1
```

Comment:
I'm not sure what this refers to, but if it's an external repo/site, can we also move it inside this repo to make things self-contained and also clear what we're using?

Reply:
I assume you refer to the mvr320/ part? its a github action which I had to fix. I dont have the rights to move it to the DOMjudge organisation but if someone does this we can directly archive it also (as I see no reason in maintaining it as DOMjudge)

if its about the downloading of the phpcs.phar, you would lose using the latest version but that could also be moved to the action or directly commited to a directory in the domjudge repo (to pin to a specific version of the sniffer)

I'm fine with all options, just not 100% sure what you want me to change.